packages/generators/remove-generator.ts

```diff
@@ -32,13 +32,15 @@ export default class RemoveGenerator extends Generator {
 				webpackOptions: {}
 			}
 		};
-
-		let configPath = path.resolve(process.cwd(), "webpack.config.js");
+		const {configFile} = opts
+		let configPath = path.resolve(process.cwd(), configFile);
 		const webpackConfigExists = fs.existsSync(configPath);
+
 		if (!webpackConfigExists) {
```

Comment:
This checking is not required now.

Reply:
@misterdev what about this one ? should I remove this if statement?

Reply:
Yeah you are right! We can remove it